.gitignore

```diff
@@ -2,6 +2,9 @@ env
 venv
 docs/_build/*
 target/
+typings/
+pyrightconfig.json
```

Comment:
My editor use pyright to link/typecheck so need to ignore some config files related to it.